docs/vectors/index.rst

```diff
@@ -0,0 +1,44 @@
+Reference vectors
+=================
+
+In addition to the policy of keeping any code path in PyNaCl covered by
+unit tests, the output from cryptographic primitives and constructions
+must be verified as really being conformant to the reference implementations
+or standards.
+
+Imported reference vectors
+--------------------------
+
+Wherever possible, PyNaCl policy is to pull already-available reference
```

Comment:
Wherever possible it is the PyNaCl project's policy to use existing reference vectors for primitives or constructions. These vectors should ideally be in their original format, but it is acceptable to make minimal changes to ease parsing at our discretion.